webrender/src/renderer.rs

```diff
@@ -1076,6 +1077,7 @@ impl Renderer {
                    target_size: &DeviceSize,
                    cache_texture: Option<TextureId>,
                    should_clear: bool) {
+        let _ = GpuMarker::new("target draw");
```

Comment:
Instead of these hard coded strings, could we use the label field that exists in the GpuProfileTag structure?